docs/xml/elements/hours_open.rst

```diff
@@ -1,57 +1,103 @@
+|                     |                  |           |          |and end times and     |implementation is required  |
+|                      options end.          |to ignore it.               |
 +---------------------+------------------+-----------+----------+----------------------+----------------------------+
 
 HoursOpen.Schedule.Hours
 ------------------------
 
-+-----------+---------------+-----------+----------+----------------------+----------------------------+
-| Tag       | Data Type     | Required? | Repeats? |Description           |Error Handling              |
-|           |               |           |          |                      |                            |
-+===========+===============+===========+==========+======================+============================+
-| StartTime | TimeWithZone  | Optional  | Single   |                      |                            |
-+-----------+---------------+-----------+----------+----------------------+----------------------------+
-| EndTime   | TimeWithZone  | Optional  | Single   |                      |                            |
-+-----------+---------------+-----------+----------+----------------------+----------------------------+
+The open and close time for this place. All times must be fully specified,
+including a timezone offset from UTC.
+
++-----------+---------------+-----------+----------+----------------------+-------------------------------+
+| Tag       | Data Type     | Required? | Repeats? |Description           |Error Handling                 |
+|           |               |           |          |                      |                               |
++===========+===============+===========+==========+======================+===============================+
+| StartTime | TimeWithZone  | Optional  | Single   |The time at which this|If the field is invalid or     |
+|           |               |           |          |place opens.          |not present, the implementation|
+|           |               |           |          |                      |is required to ignore it.      |
++-----------+---------------+-----------+----------+----------------------+-------------------------------+
+| EndTime   | TimeWithZone  | Optional  | Single   |The time at which this|If the field is invalid or not |
+|           |               |           |          |place closes.         |present, the implementation is |
+|           |               |           |          |                      |required to ignore it.         |
++-----------+---------------+-----------+----------+----------------------+-------------------------------+
 
 .. code-block:: xml
    :linenos:
 
-   <HoursOpen id="hours0000">
-     <Schedule>
-       <Hours>
-         <StartTime>08:30:00-05:00</StartTime>
-	 <EndTime>17:00:00-05:00</EndTime>
-       </Hours>
-     </Schedule>
-   </HoursOpen>
+  <HoursOpen id="hours0001">
```

Comment:
I'm so used to early vote sites that open and close when the clerk goes to get lunch... yes, -05:00 is the offset.